core/src/main/java/org/locationtech/jts/index/quadtree/Node.java

```diff
@@ -0,0 +1,183 @@
+
+/*
+ * Copyright (c) 2016 Vivid Solutions.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v20.html
+ * and the Eclipse Distribution License is available at
+ *
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+package org.locationtech.jts.index.quadtree;
+
+import org.locationtech.jts.geom.Envelope;
+import org.locationtech.jts.util.Assert;
+
+/**
+ * Represents a node of a {@link Quadtree}.  Nodes contain
+ * items which have a spatial extent corresponding to the node's position
+ * in the quadtree.
+ *
+ * @version 1.7
+ */
+public class Node
```

Comment:
While I'm still interested in identifying options that'd allow Sedona to use a JTS release without forking/copying any of the code, I'd point that using the same FQCNs may lead to classpath / classloader issues.  

Basically, if you really have to do this, I'd suggest changing the package names.  

Again, reflection may be a winning, short term solution here!